### ISSUE TITLE
Where is service-worker.js?

### ISSUE THREAD

**Issue by KingMob** [gothinkster/clojurescript-reframe-realworld-example-app#3]
It's listed in `conduit.core/main`, but I can't find any other reference to it.

**Comment by SVMBrown** [comment]
Service Workers are part of the JS Navigator API, see here:
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/serviceWorker

**Comment by KingMob** [comment]
Um, thank you for the link to MDN, but that doesn't address the question at all. I'm pointing out that this repo is missing a service-worker.js file that's referenced in the code, not asking for general info on service workers. 

In the original repo, https://github.com/jacekschae/conduit, has that line commented out, which makes more sense.

**Comment by SVMBrown** [comment]
Sorry, my mistake. Saw this issue in passing and didn't look closely enough before responding.

**Comment by dijonkitchen** [comment]
For reference, this is the LOC: https://github.com/gothinkster/clojurescript-reframe-realworld-example-app/blob/master/src/conduit/core.cljs#L79

In the original, it was commented out in this commit: https://github.com/jacekschae/conduit/commit/32f71058ab35783faecad4ef6165617de7bfe1d2#diff-d1af0725d6b980deca7e6599939011ff

**Comment by dijonkitchen** [comment]
This seems related to a root issue where the original source repository isn't synced with the one here. 

@jacekschae can you address this?

**Comment by jacekschae** [comment]
Added depreciateion warning and also did PR to swap link on the home page of gothinkster. Hope this will clarify the situation. 

@dijonkitchen thanks for the PR will make the spelling correction on the main repo

@KingMob the service-worker.js was an experiment at one point and later on I removed it from the repo. Mainly becuase of quite a bit of problems when managing SW - here is a great summary by Rich Harris https://gist.github.com/Rich-Harris/fd6c3c73e6e707e312d7c5d7d0f3b2f9
